pkg/cli/environment.go

```diff
@@ -21,6 +21,8 @@ const (
 	EnvironmentTest string = "test"
 	// EnvironmentDevelopment is the Development Environment name
 	EnvironmentDevelopment string = "development"
+	// Environmentexp is the GovCloud exp Environment name
+	Environmentexp string = "exp"
```

Comment:
We probably want to make this CamelCase just to fit the pattern:
```suggestion
	EnvironmentExp string = "exp"
```